cmd/root.go

```diff
@@ -168,9 +174,19 @@ func validateAuthentication(cmd *cobra.Command, args []string) error {
 }
 
 func ExecuteContext(ctx context.Context) error {
+	start := time.Now()
 	err := RootCmd.ExecuteContext(ctx)
 
+	sentryCommandContext := sentry_utils.NewCommandContext(start)
+	sentryCommandContext.SetOnCurrentScope()
+
 	if err == nil {
+		sentry.CaptureMessage(fmt.Sprintf("%s executed successfully", sentryCommandContext.Name))
```

Comment:
very very nice

Reply:
where is the sentryCommandContext.Name defined ?

Reply:
https://github.com/groundcover-com/cli/blob/ab5d3c73cfbb16a98e20dc83c461e5b1e080b732/cmd/root.go#L96
https://github.com/groundcover-com/cli/blob/ab5d3c73cfbb16a98e20dc83c461e5b1e080b732/pkg/sentry/context.go#L30